entrypoint.sh

```diff
@@ -16,3 +16,5 @@ deployer:
   name: "${INPUT_DEPLOYER_NAME:-${GITHUB_ACTOR}}"
   email: "${INPUT_DEPLOYER_EMAIL}"
 EOF
+
+opslevel create deploy -i "${INPUT_INTEGRATION_URL}" -f .
```

Comment:
`-f .` tells the CLI to read input from the `data.yaml` file - https://github.com/OpsLevel/cli/blob/9d06f70df2bd7d3ea700cd8eec72b8dff3680dfb/src/cmd/create.go#L21.